### DISCUSSION TITLE
Venus port (for 2.0.1+)

### DISCUSSION THREAD

**Discussion by Jigoku** [redeclipse/base#1350] [Maps and Content Showcase]
Venus ported to 2.0+ (needs current dev branch for haze / other features)

Changes since the last 1.6 version:
* increase geometry detail everywhere
* redone texturing / lights / blendmap / materials / palette / waypoints
* removed 2 grenade spawns (previously there were 2 on each bridge)
* use models for waterfall (some issues with this, see below)
* added some alien egg models
* added some stalagmite/stalactite models
* additional texture added for some blendmap details
* team base changed on exterior appearance
* enlarged the small rooms for bomber modes
* has a map variant (although the 2nd one may need more thought put into it)

Download: https://ufile.io/qukok0g0 (place the zip in the root of your REDECLIPSE_HOME folder)

The waterfalls are using the nieb mapmodels, with waterfall material behind them, although there's a rendering artifact where skymat(?) is seen on the underside of the top piece of the water. Still trying to think how to make this look better :confused: 

There's more than likely room for some improvements still.

Some Screenshots:
![2022-06-05-001241_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028702-938e2ad0-273a-4bcc-bb95-635b2f7b4868.png)
![2022-06-05-001304_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028704-66916b67-fa51-44ca-a15b-d7ebc9df6d2e.png)
![2022-06-05-001315_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028705-56960473-463a-4574-99f3-c9920498a01b.png)
![2022-06-05-001332_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028706-1b96f52e-c45d-407b-a50a-dce228a92ac7.png)
![2022-06-05-001347_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028707-3b155f2a-a7a3-43bc-8cbe-6747184a68e1.png)
![2022-06-05-001412_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028710-31b4f66c-dbad-4161-bc9e-68a15c726d75.png)
![2022-06-05-001419_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028711-aecde0af-0183-4e64-9ad9-e3cabad2e30a.png)
![2022-06-05-001528_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/172028719-7298e55b-9397-40b2-be2b-06552995627a.png)

**Comment by qreeves** [comment]
Most of the included content in this should probably go in the `misc` submodule.

**Reply by Jigoku** [reply]
Yup, will do that when opening a PR. The folder structure at the moment is just what i was dealing with locally. Also, i'll try play more with those waterfalls at the top to see if they can be improved more soon. :+1:

**Comment by MirceaKitsune** [comment]
Old download link is broken: Is there a new one we can try this out at? The map looks absolutely amazing!

**Reply by qreeves** [reply]
Yeah @Jigoku - you should upload them as an attachment on GitHub rather than externally host these 👍

**Reply by Jigoku** [reply]
yeah my bad, github is restricted to 50mb for files on issue/comments; i'll upload it somewhere else later :+1:

**Reply by Jigoku** [reply]
It's just been merged into master, so you can test it out there

**Reply by MirceaKitsune** [reply]
Played a few matches with bots and it works perfectly. Thank you for porting one of the best outdoor maps currently available!
